### DISCUSSION TITLE
How to set up storybook and nativewind

### DISCUSSION THREAD

**Discussion by dohomi** [nativewind/nativewind#190] [Q&A] [answered]
Hey Guys,

I am trying to set up storybook in combination with the Solito starter and nativewind.

Did anyone successfully get this up and running? I am using the `@storybook/addon-react-native-web` plugin but a story is not running due to the error:
```
TypeError: Cannot read properties of undefined (reading 'create')
    at ../../node_modules/nativewind/dist/style-sheet/runtime.js
```
It would be great to have monorepo example which renders stories with nativewind

**Comment by dohomi** [comment]
@marklawlor I created a kitchen-sink to showcase several apps running with nativewind. Storybook with react is initialised here: 
https://github.com/dohomi/nativewind-solito-kitchen-sink/blob/master/apps/storybook-react/.storybook/main.ts#L19
Do you have experience in how this setup would look like?

**Comment by dohomi** [comment, marked as answer]
Ok I updated the kitchen sink and enabled storybook-react, storybook-react-native and jest. Hope this will help anybody else:
[https://github.com/dohomi/nativewind-solito-kitchen-sink](https://github.com/dohomi/nativewind-solito-kitchen-sink/blob/master/apps/storybook-react/.storybook/main.ts#L19)

**Reply by dannyhw** [reply]
This is awesome! I was trying to make it work with @storybook/addon-postcss instead of adding the loader to webpack directly but had some trouble. Have you tried with the storybook postcss addon?

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
